adr/0003-rds-permissions.md

```diff
@@ -0,0 +1,44 @@
+# AWS RDS permissions
```

Comment:
lIf this is only related to Postgresql, it can be named `rds-postgresq-permissions`?